asserts/store_asserts.go

```diff
@@ -0,0 +1,61 @@
+package asserts
+
+import "fmt"
+
+// EnterpriseStore holds an enterprise-store assertion, defining the
+// configuration needed to connect a device to the enterprise store.
+type EnterpriseStore struct {
+	assertionBase
+}
+
+// OperatorID returns the account id of the enterprise store's operator.
+func (estore *EnterpriseStore) OperatorID() string {
+	return estore.HeaderString("operator-id")
+}
+
+// Store returns the name of the operator's enterprise store.
```

Comment:
Not just for display - it's part of the primary key so identifies an operator's specific store in case there are 2+.